CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contribution Guidelines
+
+## Infrastructure
+
+We have two Airflow servers:
+
+<!-- Do we need `airflow-dev` if we have DevContainers? -->
+1. `airflow-dev`: Hosted in the `dnt-dev` AWS account. Deployed manually. Has a public IP address (accessible using the VPN).
```

Comment:
Eventually, can this be a non-manual deployment using CI/CD?

Reply:
A follow-up PR will automate the deployment.